### PR TITLE
Issue 3 nys teacher evaluation databases

### PR DESCRIPTION
**Pull request by zachary-sproul**
Feel free to rip me apart and say that this is bad.

I basically just created functionality to receive a ZIP file, find the Access db (mdb) inside of it, copy it to a temp location, and read from it.

1) My research indicates that we can't read from access databases in zip files.
2) I'm fine with replacing OleDbConnection with a specialized nuget for mdb files.
3) I'm sure that the IngestBase was made in a specific way on purpose and I ruined it.

### CODE REVIEW

File: CensusAnalytics/OpenDataLoader/ODL.Common/IngestBase.cs [hunk 1]
```diff
@@ -7,10 +7,11 @@
 
 namespace ODL.Common
 {
-    internal abstract class IngestBase
+    public abstract class IngestBase
```
Comment (zachary-sproul) [inline CensusAnalytics/OpenDataLoader/ODL.Common/IngestBase.cs:10]: I did this so my command line program would work. My understanding is that I can't access this or implementing classes unless they are public.

File: CensusAnalytics/OpenDataLoader/ODL.Common/IngestImplementations/TeacherEvaluations.cs [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                                }
+
+                                foreach (var tableName in tableNames)
+                                {
+                                    Console.WriteLine(tableName);
+                                }
+                            }
+                            catch (Exception ex)
+                            {
+                                Console.WriteLine(ex.Message);
```
Comment (zachary-sproul) [inline CensusAnalytics/OpenDataLoader/ODL.Common/IngestImplementations/TeacherEvaluations.cs]: I know this is where the logger is to be used

File: CensusAnalytics/OpenDataLoader/ODL.Common/IngestImplementations/TeacherEvaluations.cs [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+                            }
+                        }
+
+                        File.Delete(tempPath);
+                    }
+                }
+            }
+
+            throw new NotImplementedException();
```
Comment (zachary-sproul) [inline CensusAnalytics/OpenDataLoader/ODL.Common/IngestImplementations/TeacherEvaluations.cs]: Yep, not done yet

**Comment by jdyelle** [comment]
I find it unfortunate that there aren't any acceptable .net standard libraries that will allow reading an access DB -- I did a little reasearch and I think your approach is likely the most reasonable way to go.

**Pull request merged**